_sass/home.scss

```diff
@@ -43,10 +45,14 @@
         flex   : 1;
       }
 
-      div {
-        flex : 2;
+      .regularText {
+        color: #1e1e1e;
       }
     }
+    //a:hover {
+    //  outline: dashed 1px #1e1e1e; /* uncomment for dashed border */
+    //}
+

```

Comment:
Remove this pice of commented style, no need to add a dashed border when hovering over the buttons